source/diagrams/manage-diagrams/delete-diagrams.txt

```diff
@@ -10,4 +10,63 @@ Delete Diagrams
    :depth: 1
    :class: singlecol
 
-Delete diagrams.
\ No newline at end of file
+You can delete diagrams on the :guilabel:`Mapping` tab of Relational
+Migrator. 
+
+About this Task 
+---------------
+
+- You must have at least two diagrams before you can delete a diagram
```

Comment:
[suggestion] I'd reverse the order here for clarity. Also, there's no need for a single item to be in an unordered list. 

"Relational Migrator requires at least one diagram in your project. You must have at least two diagrams before you can delete one diagram."

Reply:
Done!